src/plugin.js

```diff
@@ -1,6 +1,8 @@
+import 'babel-polyfill';
```

Comment:
What is babel-polyfill needed for here? Since this is a library, we generally prefer not including polyfills, requesting users include it on their page themselves.
Also, should this be using `@babel/polyfill` as it's newer?

Reply:
This seemed to be a dependency of webxr polyfill - I'll recheck

Reply:
Seems like it isn't required https://github.com/immersive-web/webxr-polyfill#browser-support

I think we should make a note of it in the README, but otherwise leave it out of the bundle.

Reply:
Ok so I tried pulling it out this results in the WebXR sessions not starting because there is some regenerator code not generated without babel-polyfill
I encourage other to help remove it if its a concern, I'm out of cycles today

Reply:
going to "unresolve" this just so that it doesn't get lost, but I don't think it's a blocker.

Reply:
FWIW, when I test this on oculus browser 36 / chromium 130, this works well. I would remove the polyfill here and leave it up to consumers if they want to support older browsers?